packages/cli/src/cli/utilities/help.ts

```diff
@@ -0,0 +1,33 @@
+import {Command, Help, CommandHelp} from '@oclif/core'
+import chalk from 'chalk'
+
+const {dim} = chalk
+
+class CustomCommandHelp extends CommandHelp {
+  protected override flags(flags: Command.Flag.Any[]): [string, string | undefined][] | undefined {
+    if (flags.length === 0) return
+
+    return flags.map((flag) => {
+      const left = this.flagHelpLabel(flag)
+
+      let right = flag.summary || flag.description || ''
```

Comment:
nit: the naming's a bit short and it's a bit hard to tell at first glance what this is referring to, maybe `left_column` and `right_column` could help make things more clear?